src/main/java/org/owasp/wrongsecrets/securityconfig/ActuatorSecurityConfiguration.java

```diff
@@ -1,25 +0,0 @@
-package org.owasp.wrongsecrets.securityconfig;
-
-import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
-import org.springframework.core.annotation.Order;
-import org.springframework.security.config.annotation.web.builders.HttpSecurity;
-import org.springframework.security.web.SecurityFilterChain;
-
-/**
- * Configuration used to disable CSRF on the actuator health endpoint otherwise it does not work for health-checks.
- */
-@Configuration
-public class ActuatorSecurityConfiguration {
```

Comment:
hmmm this is something I would love to test. because I did not get it to work for the CTF setup without this for some dark reason... I will build a container from your branch and try it :D , will get back to you on this.

Reply:
The health checks are GET methods so they shouldn't need CSRF (would otherwise not work in a Kubernetes environment)